settings.gradle

```diff
@@ -58,6 +58,17 @@ gradleEnterprise {
     }
 }
 
+buildCache {
+    remote(HttpBuildCache) {
+        url = 'https://ge.armeria.dev/cache/armeria/'
+        credentials { creds ->
+            creds.username = System.getenv('GRADLE_ENTERPRISE_CACHE_USERNAME')
+            creds.password = System.getenv('GRADLE_ENTERPRISE_CACHE_PASSWORD')
+        }
+        push = System.getenv('CI') != null
```

Comment:
If you use access key authentication there's no need to set up specific credentials for the remote cache since it's managed by gradle enterprise. You could also reuse the `isCI` variable to enable pushing to remote cache. An example of a simple setup can be seen [here](https://github.com/gradle/android-cache-fix-gradle-plugin/blob/main/settings.gradle.kts)

Reply:
I see, the reason I used `GRADLE_ENTERPRISE_CACHE_USERNAME` was specifically because I couldn't get it working with the `ci-agent` role. Let me give it another try 😅

Reply:
Removed the cache username/password credentials and triggered a run.
Hopefully it works 🤞 
- https://github.com/jrhee17/armeria/pull/27
- https://github.com/line/armeria/compare/main...jrhee17:armeria:main

Reply:
You do need to configure the access key though:

> In general, we'd suggest to use [access key authentication](https://docs.gradle.com/enterprise/gradle-plugin/#manual_access_key_configuration), which is simpler to set up both on CI and locally as for local authentication your developers can just run ./gradlew provisionGradleEnterpriseAccessKey to authenticate automatically.
> 
> For CI setup you can observe an example [here](https://github.com/gradle/common-custom-user-data-gradle-plugin/blob/55350a142f9ddfc847e986929aa33938719ce9cb/.github/workflows/build-verification.yml#L22).

Reply:
The credentials are configured correctly.